src/main.rs

```diff
@@ -50,17 +44,17 @@ fn main() ->  Result<(), Err()> {
     });
 
     // Send the email
-    let response = Client::new()
-        .default_headers(headers)
+    let client = reqwest::blocking::Client::new();
+    let response = client.post("https://api.sendgrid.com/v3/mail/send")
         .json(&body)
-        .post("https://api.sendgrid.com/v3/mail/send")
-        .send().await?;
+        .basic_auth("",Some(auth_token))
```

Comment:
@settermjd I have no Idea if this works correctly as I do not know what `basic_auth` does with the username and password. As we should send a bearer token and no username and Passwort it is technically not basic auth, so maybe this still does not work.

Reply:
To bad, just checked the implementation and this will definitely not work :disappointed: 
But I found a way to inject request headers :smile:

Reply:
All good. Between your PR and the other one I received, I'll get it going. Really appreciate your time and feedback.